src/components/editor/faultTree/canvas/EditorCanvas.tsx

```diff
@@ -60,9 +64,24 @@ const EditorCanvas = ({
   const [svgZoom, setSvgZoom] = useState(null);
   const [currentZoom, setCurrentZoom] = useState(1);
   const [isExportingImage, setIsExportingImage] = useState(false);
+  const [targets, setTargets] = useState<undefined | string[]>();
 
   let dragStartPosition = null;
 
+  useEffect(() => {
+    const targetsIri = [];
+
+    faultEventScenarios.forEach((scenario) => {
+      if (Array.isArray(scenario?.scenarioParts)) {
+        scenario.scenarioParts.forEach((sp) => targetsIri.push(sp.iri));
```

Comment:
`sp` can be renamed `scenarioPart` to make it more explicit

Reply:
Changed